### PR TITLE
Update bindgen version & fix some cache issue

### LINKED ISSUE #447: Not reusing cache in test
## 🐛 Bug description
when I run the "test build::it_should_build_js_hello_world_example", I notice that it reinstall wasm-bindgen for cargo while it had downloaded  prebuilt wasm-bindgen.

In fact, in the test it had installed the `wasm-bindgen` and the `wasm-bindgen-test-runner`  binaries so it should just copy them to the fixture directory, and using them, no need to re install
```
#[test]
fn it_should_build_js_hello_world_example() {
    let fixture = utils::fixture::js_hello_world();
    fixture.install_local_wasm_bindgen();   // we installed first before begin the test!
    let cli = Cli::from_iter_safe(vec![
        "wasm-pack",
        "build",
        &fixture.path.display().to_string(),
    ])
    .unwrap();
    fixture.run(cli.cmd).unwrap();
}

```
#### 🤔 Expected Behavior
It should copy the local wasm-bindgen in test_cache

#### 👟 Steps to reproduce
1. cargo clean
2. click the single test in vs code of `it_should_build_js_hello_world_example`, so that you can see the log
3. wait for compilation. It should show the log as follow:
```
running 1 test
Created fixture at c:\Users\34937\Documents\wasm-pack\target\t\.tmp6BHDQ1\wasm-pack

  [1/9] Checking `rustc` version...
  [2/9] Checking crate configuration...
  [3/9] Adding WASM target...
  info: component 'rust-std' for target 'wasm32-unknown-unknown' is up to date
  [4/9] Compiling to WASM...
      Updating `git://mirrors.ustc.edu.cn/crates.io-index` index
     Compiling proc-macro2 v0.4.20
     Compiling unicode-xid v0.1.0
     Compiling serde v1.0.80
     Compiling ryu v0.2.6
     Compiling wasm-bindgen-shared v0.2.21
     Compiling itoa v0.4.3
     Compiling cfg-if v0.1.6
     Compiling lazy_static v1.2.0
     Compiling wasm-bindgen v0.2.21
     Compiling log v0.4.6
     Compiling quote v0.6.9
|    Compiling syn v0.15.18
test build::it_should_build_js_hello_world_example ... test build::it_should_build_js_hello_world_example has been running for over 60 seco     Compiling syn v0.15.18
     Compiling serde_json v1.0.32
     Compiling serde_derive v1.0.80
     Compiling wasm-bindgen-backend v0.2.21
     Compiling wasm-bindgen-macro-support v0.2.21
     Compiling wasm-bindgen-macro v0.2.21
     Compiling js-hello-world v0.1.0 (C:\Users\34937\Documents\wasm-pack\target\t\.tmp6BHDQ1\wasm-pack)
      Finished release [optimized] target(s) in 4m 59s
  [5/9] Creating a pkg directory...
  [6/9] Writing a package.json...
  [7/9] Copying over your README...
  [8/9] Installing wasm-bindgen...
  [9/9] Running WASM-bindgen...
  :-) Done in 5 minutes
| :-) Your wasm pkg is ready to publish at "\\\\?\\C:\\Users\\34937\\Documents\\wasm-pack\\target\\t\\.tmp6BHDQ1\\wasm-pack\\pkg".
test build::it_should_build_js_hello_world_example ... ok

test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 41 filtered out
```

#### 🌍 Your environment
Include the relevant details of your environment.
wasm-pack version:  github master branch, commit hash 51e6351 
rustc version: rustc 1.32.0-nightly (6e9b84296 2018-11-10)

### LINKED ISSUE #519: bump up the wasm-bindgen version in test?
## 💡 Feature description
I notice that different test use a different version of  wasm-bindgen, I think we can use the same version(I mean always the newest version) of wasm-bindgen, so that ALL THE TEST would use the same cache

#### 💻 Basic example
the test that use fixture has locked the `wasm-bindgen` and `wasm-bindgen-test` to version `0.2.21`,(for example, `it_should_build_js_hello_world_example`) while other test that generates its fixture (for exmaple, `build_with_and_without_wasm_bindgen_debug`) may use just the auto-patching version config of `0.2`.
that means we have to download at least 2 version of wasm-bindgen cache for testing, and that spends a lot of time (especially for worker in country with network inspection where slow down github).

but if we bump up the `wasm-bindgen` version of fixture to `0.2`, we can use the same cache and will only fetch `wasm-bindgen` only one time.

related to #447

### PR DESCRIPTION
**Pull request by huangjj27**
Make sure these boxes are checked! 📦✅

- [x] You have the latest version of `rustfmt` installed
- [x] You ran `cargo fmt` on the code base before submitting
- [x] You reference which issue is being closed in the PR text

closes #519  and fixes #447

### CODE REVIEW

**Comment by huangjj27** [comment]
the failling test printed test output at both [stdout](https://travis-ci.com/rustwasm/wasm-pack/jobs/173192596#L2817) and [stderr](https://travis-ci.com/rustwasm/wasm-pack/jobs/173192596#L2827). where the stdout is not expected.

File: CHANGELOG.md [hunk 1]
```diff
@@ -32,7 +32,11 @@
 
     Currently, you can only configure things related to the above-mentioned build profiles. To learn more, 
     [check out the documentation][profile-config-docs]. It leverages the `package.metadata.wasm-pack` key in your
+<<<<<<< HEAD
```
Comment (csmoe) [inline CHANGELOG.md]: rebasing seems not finished yet.

Comment (huangjj27) [reply CHANGELOG.md]: changelog is checked and fixed. Any issue else?

Comment (csmoe) [reply CHANGELOG.md]: no more :)

**Comment by ashleygwilliams** [comment]
@huangjj27 is this still a work in progress? we should figure out that failing test- lemme know if you want help! i'll make as a WIP until i hear back from you

**Comment by huangjj27** [comment]
I don’t think the failing test caused by, my PR, as I only change the wasm-bindgen version.

**Comment by huangjj27** [comment]
@ashleygwilliams I think the PR is ready，and @csmoe has reviewed my code，please check and merge

**Comment by ashleygwilliams** [comment]
closing in favor of https://github.com/rustwasm/wasm-pack/pull/554

**Pull request closed without merging**